### DISCUSSION TITLE
Team discussions deprecated

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#97] [Team Posts]
Oops. 

<img width="895" alt="Screen Shot 2023-02-17 at 2 36 05 PM" src="https://user-images.githubusercontent.com/10963114/219666900-d379e5f0-60d2-4c1e-bc9a-899fbc7d871c.png">

Will we still be able to have team discussions essentially? Like tagging a team in a GitHub discussion topic?

**Comment by arademaker** [comment]
it makes sense. I was about to suggest we need to start using the discussions.

**Comment by arademaker** [comment]
We now have the organization discussions
https://github.com/orgs/delph-in/discussions

and also the 
https://github.com/delph-in/docs/discussions

I believe everything in the first link is also available in the second link. What I don't know yet is what happen with the visibility of the messages!

**Reply by EricZinda** [reply]
I *think* that you have to enable repository level discussions in a repository like docs to have a more local area for discussion.  Right now it is just pointing at the global delph-in discussion by default. 

Can one of you try it so we can put our docs discussions there?  You just go to the repository and click on the settings and enable it.

**Reply by arademaker** [reply]
When we create an organization-level discussion, GitHub must choose one repository to hold it. It is natural to use the `docs` repository. Moreover, is it necessary to have too many different discussion forums? I don't think so. But waiting more comments.

**Reply by EricZinda** [reply]
The problem is that, currently, you can only get notifications about new topics for an *entire repository*. So, if you want to get notified when discussions happen in the docs working group, you have to subscribe to *all* notifications in the entire DELPH-IN org. The converse is also true: all of DELPH-IN (that registered for notifications) will get notified whenever the doc working group posts.  There is a discussion about adding a feature to get notifications for a category in the discussion (like the [docs](https://github.com/orgs/delph-in/discussions/categories/docs-working-group) one I added) but it doesn't have any [recent news](https://github.com/community/community/discussions/3951), so who knows when it will happen.

So, it seems like the docs repository should just have a discussion area for just docs discussions, and the delphin org should have its own, different, repository for global issues.  Worst case we could create a "fake" repository for the global discussions.

**Reply by olzama** [reply]
Maybe it's ok to have all discussions in one place, even if it is the working group? People can just ignore whatever is not relevant to them at the moment? Not sure though.

**Reply by arademaker** [reply]
I agree @olzama.
